src/fr/bluesolo/build.gradle

```
@@ -0,0 +1,10 @@
+ext {
+    extName = 'Blue Solo'
+    extClass = '.BlueSolo'
+    themePkg = 'pizzareader'
+    baseUrl = 'https://bluesolo.org/'
+    overrideVersionCode = 1
```

Use the theme version as the initial version

```suggestion
    overrideVersionCode = 0
```